e2e/cypress/integration/search_date_filter/string_spec.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getAdminAccount} from '../../support/env';
+
+import {
+    searchAndValidate,
+    getTestMessages,
+    setupTestData,
+} from '../search_filter/helpers';
+
+describe('Search Date Filter', () => {
+    const testData = getTestMessages();
+    const {commonText} = testData;
+    const admin = getAdminAccount();
+    let anotherAdmin;
+
+    before(() => {
+        cy.apiInitSetup({userPrefix: 'other-admin'}).then(({team, user}) => {
+            anotherAdmin = user;
+
+            // # Visit town-square
+            cy.visit(`/${team.name}/channels/town-square`);
+
+            setupTestData(testData, {team, admin, anotherAdmin});
+        });
+    });
+
+    it('MM-T603 Place a string when a date is expected', () => {
+        searchAndValidate(`on:hippo ${commonText}`, []);
+        searchAndValidate(`before:hippo ${commonText}`, []);
+        searchAndValidate(`after:hippo ${commonText}`, []);
+    });
```

Comment:
I'd suggest to move this into `e2e/cypress/integration/search_filter/mixed_spec.js` so that it could use the same test data and no need to generate in other place.

Reply:
Fixed it.
I first put it in that directory because the issue said so.

I must add that other 2 existing tests in `mixed_spec.js` fail.

Reply:
Thanks for updating and noted on the failed tests. It's a known issue due to the recent change.